### ISSUE TITLE
Option to set colors via hex code

### ISSUE THREAD

**Issue by Two-Tone** [easyfuckingpeasy/chrome-reddit-comment-highlights#8]
I feel like this had been an option at one time, but I can't say for certain.

An option to set the color of quotes would be nice too, but that may start end up being too much.

**Comment by aesy** [comment]
Yeah you're right, that disappeared when we dropped jQuery and had to switch color picker. I agree it would be very nice to have, but I haven't found any decent and tiny color picker with text inputs, but I'll look into it some more.

Edit: regarding the quotes, I think that's covered by the custom CSS feature. You should be able to do something like `blockquote { color: red; }`. Don't want to clutter the standard options page all too much!

**Comment by Two-Tone** [comment]
Hmm, any way to write a custom textbox that updates the relevant color
picker?

Yeah, I had a feeling doing quotes would be too much.

**Comment by aesy** [comment]
That could be done, but I would prefer a picker that supports it rather than hacking it in. So I tried switching to the native color picker and it looks pretty good and it's one less dependency to worry about! I also added ability to change the link & quote text color, but it's hidden behind 'advanced options'! I need to test it (using the very reliable method of test-by-usage) before I push it to chrome web store, so it may take a week until you see the changes :)
